pkg/vm/engine/tae/tables/updates/delchain.go

```diff
@@ -29,6 +29,18 @@ import (
 	"github.com/matrixorigin/matrixone/pkg/vm/engine/tae/wal"
 )
 
+func mockTxn() *txnbase.Txn {
+	txn := new(txnbase.Txn)
+	txn.TxnCtx = txnbase.NewTxnCtx(common.NewTxnIDAllocator().Alloc(), types.NextGlobalTsForTest(), types.TS{})
+	return txn
+}
+
+func MockTxnWithStartTS(ts types.TS) *txnbase.Txn {
```

Comment:
If MockTxnXXX is a common requirement in the unit test, we can add some mock txn functions in the related module